cypress.config.ts

```diff
@@ -2,6 +2,8 @@
 import { defineConfig } from 'cypress';
 
 export default defineConfig({
+  viewportWidth: 1200,
+  viewportHeight: 720,
```

Comment:
does this keep sneaking in because of your local setup? Or is this on purpose?

Reply:
It was added on purpose.